lib/omniauth/strategies/scalingo.rb

```diff
@@ -23,32 +23,32 @@ def authorize_params
         end
       end
 
-      uid { raw_info['id'].to_s }
+      uid { raw_info["id"].to_s }
 
       info do
         {
-          'uuid'       => raw_info['id'],
-          'username'   => raw_info['username'],
-          'email'      => raw_info['email'],
-          'first_name' => raw_info['first_name'],
-          'last_name'  => raw_info['last_name'],
-          'flags'      => raw_info['flags'],
+          "uuid" => raw_info["id"],
+          "username" => raw_info["username"],
+          "email" => raw_info["email"],
+          "first_name" => raw_info["first_name"],
+          "last_name" => raw_info["last_name"],
+          "flags" => raw_info["flags"]
         }
       end
 
       extra do
-        {:raw_info => raw_info}
+        {raw_info: raw_info}
       end
 
       def raw_info
-        @raw_info ||= access_token.get('/v1/users/self').parsed['user']
+        @raw_info ||= access_token.get("/v1/users/self").parsed["user"]
       end
 
       def callback_url
-        full_host + script_name + callback_path
+        full_host + callback_path
```

Comment:
I see this change here, what's the impact?

Reply:
omniauth 2+ respects script_name, cf the bottom of the release notes : https://github.com/omniauth/omniauth/releases/tag/v2.0.0